### DISCUSSION TITLE
Guidlines & Link Format

### DISCUSSION THREAD

**Discussion by john-ostrom** [WalletConnect/walletconnect-monorepo#1093] [Developer Support]
Hello,

Some softball baseline questions here...

Is there a way to register the app before a release for testing purposes? Do you have any suggestions for this issue. I'm trying to follow the registry guidelines but it says the app must be deployed. I'm assuming the registry process is going to fill out the session key in the deep link or universal url. For instance, when I use the example dapp everytime the key in the url is different. Thus, I cannot give testers a url to use.

I just wanted to verify what I read in another thread. When filling out the form for the wc registry the deep link url is formatted like:  "mydeeplinkid:"

The follow up question, do you have an example of a universal link as it would appear exactly in the registry application form? I don't want to guess what to submit. If we messed up the format would wc contact us to fix it?

Thanks for your patience.

**Comment by chadyj** [comment]
> The follow up question, do you have an example of a universal link as it would appear exactly in the registry application form? I don't want to guess what to submit. If we messed up the format would wc contact us to fix it?

You can take a look at other values here https://registry.walletconnect.com/api/v1/wallets
